### PR TITLE
add regression test

### PR DESCRIPTION
**Pull request by limeng120**
## Description

What is the purpose of the change?
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Brief of your solution

Please describe your solution to solve the issue or feature request.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

### CODE REVIEW

File: .github/workflows/regression1.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+on: pull_request
+name: TEST1
+jobs:
+  test:
+    strategy:
+      matrix:
+        python-version: ["3.7"]
+        go-version: [1.13.x]
```
Comment (zhugelianglongming) [inline .github/workflows/regression1.yml:8]: 后续可能需要相应更新到 1.16 1.18+

**Comment by godeamon** [comment]
这三个文件名字改一下吧， 1、2、3 给成功能什么的吧。

File: .github/workflows/regression1.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+on: pull_request
+name: regression_test_1
```
Comment (zhugelianglongming) [inline .github/workflows/regression1.yml:2]: 建议注释说明拆分 patch 的原因、背景、经验

File: .github/workflows/regression1.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    - name: Download testcase code
+      run: |
+        git clone https://github.com/xuperchain/regression-pytest.git
+        cd regression-pytest
+        python -m pip install --upgrade pip
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Deploy xchainnet for test
+      run: |
+        make && make testnet && cd testnet/
+        sed -i' ' 's#level: debug#level: warn#g' node1/conf/log.yaml
```
Comment (zhugelianglongming) [inline .github/workflows/regression1.yml:31]: sed -i' ' 's#level: debug#level: warn#g' node*/conf/log.yaml

File: .github/workflows/regression2.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    - name: Download testcase code
+      run: |
+        git clone https://github.com/xuperchain/regression-pytest.git
+        cd regression-pytest
+        python -m pip install --upgrade pip
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Deploy xchainnet for test
+      run: |
+        make && make testnet && cd testnet/
+        sed -i' ' 's#level: debug#level: warn#g' node1/conf/log.yaml
```
Comment (zhugelianglongming) [inline .github/workflows/regression2.yml:31]: 同上

File: .github/workflows/regression3.yml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    - name: Download testcase code
+      run: |
+        git clone https://github.com/xuperchain/regression-pytest.git
+        cd regression-pytest
+        python -m pip install --upgrade pip
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Deploy xchainnet for test
+      run: |
+        make && make testnet && cd testnet/
+        sed -i' ' 's#level: debug#level: warn#g' node1/conf/log.yaml
```
Comment (zhugelianglongming) [inline .github/workflows/regression3.yml:31]: 同上

**Review by zhugelianglongming — APPROVED** [review]

**Review by godeamon — APPROVED** [review]

**Pull request merged**